app/models/service/aggregation.rb

```diff
@@ -315,7 +315,7 @@ def aggregate_all_vm_disk_space_used
   def aggregate_all_vm_memory_on_disk
     if has_attribute?("aggregate_all_vm_memory_on_disk")
       # Avoids (poorly) "ERROR:  integer out of range" in postgres
-      self["aggregate_all_vm_memory_on_disk"] * 1.megabyte
+      self["aggregate_all_vm_memory_on_disk"].try(:*, 1.megabyte)
```

Comment:
@bdunne it might be memory for a vm that's stored in a physical file.  I think in vmware it was a .vram file?  You can probably check the vmware provider source to see what populates that column.